src/EMS/RingdownStatus.js

```diff
@@ -7,15 +7,28 @@ import Alert from '../Components/Alert';
 
 function RingdownStatus({ className, onStatusChange, ringdown }) {
   const [showCancel, setShowCancel] = useState(false);
+  const [confirmCancel, setShowConfirmCancel] = useState(false);
```

Comment:
if the state var is `confirmCancel` then the setter function can just be `setConfirmCancel`...